src/DropInViewer.js

```diff
@@ -28,6 +27,10 @@ export class DropInViewer extends THREE.Group {
       this.viewer,
     );
 
+    this.unprojectMousePosition = this.unprojectPositionFromSplats.bind(this);
```

Comment:
would keep the name here


```suggestion
    this.unprojectPositionFromSplats = this.unprojectPositionFromSplats.bind(this);
```